apps/Notepad ++/install

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+version=8.5.3
+
+#determine filename of download
+if [ $arch == 32 ];then
+  filename="https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v${version}/npp.${version}.portable.zip"
+elif [ $arch == 64 ];then
+  filename="https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v${version}/npp.${version}.portable.x64.zip"
+fi
+
+#Checking if wine is installed
+if ! command -v wine &>/dev/null ; then
+  echo 'Wine Is Not Installed Yet'
+  #Installing wine
+  if [ $arch == 32 ];then
+    "$DIRECTORY/manage" install 'Wine (x86)' || error 'Failed to install wine'
+  elif [ $arch == 64 ];then
+    "$DIRECTORY/manage" install 'Wine (x64)' || error 'Failed to install wine'
+  fi
+fi
```

Comment:
I suggest replacing this with `install-if-not-installed`. Otherwise, NPP would silently fail to launch on systems where the user already has wine installed from elsewhere. Chances are, if wine is installed from elsewhere, it is either an arm version from the repos, or a franken wine install from following a tutorial.

Reply:
this was copied from the install-32. this can be implemented

Reply:
https://github.com/Botspot/pi-apps/pull/2341/commits/d29b3114c6b4d645a618eb0a84baa74897d6c569